cms/test_utils/project/app_with_cms_feature/cms_config.py

```diff
@@ -0,0 +1,31 @@
+from cms.app_base import CMSAppExtension
+
+
+def some_function(*args, **kwargs):
```

Comment:
this is a bit odd. what does this test?

Reply:
As the comment in the next line says - that autodiscover doesn't pick this up :) Just wanted to add some random functions and classes in there, so that we know that autodiscover only ever picks up classes that inherit from CMSAppConfig and CMSAppExtension. Possibly now a bit redundant because the separation into two classes means that the code would have to check the subclass anyway, which wasn't true before.

Reply:
Yeah I think is fine to remove this